src/learn/css/Result.css

```
@@ -0,0 +1,114 @@
+body {
+  margin: 0;
+}
+
+#result {
+  display: grid;
+  grid-template:
+    "... ....... ....... ....... ..." 3vh
+    "... ....... title   ....... ..." 10vh
+    "... ....... ....... ....... ..." 2vh
+    "... ....... image   ....... ..." 60vh
+    "... ....... ....... ....... ..." 3vh
+    "... players players players ..." 80vh
+    "... ....... ....... ....... ..." 3vh
+    /20vh auto 90vh auto 20vh;
+  height: 100%;
+  width: 100%;
+}
+
+.title {
+  font-size: 8vh;
+  grid-area: title;
+  text-align: center;
+}
+
+img {
+  height: 100%;
+  width: 100%;
+}
+
+.explanationImg {
+  grid-area: image;
+}
+
+.explanationImg img {
+  object-fit: contain;
+}
+
+.players {
+  display: flex;
+  flex-direction: column;
+  grid-area: players;
+  justify-content: space-between;
+}
+
+.player {
+  display: flex;
+  flex-direction: row;
+  margin-top: 3vh;
+}
+
+.playerImg {
+  display: flex;
+  flex-basis: 20%;
+  flex-direction: column;
+}
+
+.playerImg img {
+  border-radius: 50%;
+  height: 20vh;
+  margin: auto;
+  width: 20vh;
+}
+
+.playerName {
+  flex-basis: 10%;
+  text-align: center;
+}
+
+.playerExplanation {
+  flex-basis: 80%;
+  position: relative;
+}
+
+.playerExplanation p {
+  margin: 0 0 0 2vh;
+  position: absolute;
+  top: 50%;
+  transform: translate(0, -50%);
+}
+
+.balloon {
+  border: solid 0.5vh black;
+  border-radius: 2vh;
+  box-sizing: border-box;
+  display: inline-block;
+  font-size: 4vh;
+  height: 22vh;
+  margin-left: 5vh;
+  position: relative;
+  width: 30vh;
+}
+
+.balloon::before {
+  border: 12px solid transparent;
+  border-right: 12px solid #fff;
+  content: "";
+  left: -24px;
+  margin-top: -12px;
+  position: absolute;
+  top: 50%;
+  z-index: 2;
+}
+
+.balloon::after {
+  border: 14px solid transparent;
+  border-right: 14px solid #555;
```

```suggestion
  border-right: 14px solid #black;
```

飛び出してる部分が謎の色してるからどっちかに揃えてほしい